test.js

```diff
@@ -241,10 +241,11 @@ Object.entries(POLYFILLS).forEach(([name, module]) => {
 
           class SubClass extends BaseClass {
             static getDerivedStateFromProps(nextProps, prevState) {
-              return {
-                ...BaseClass.getDerivedStateFromProps(nextProps, prevState),
-                bar: 'bar',
-              };
+              return Object.assign(
```

Comment:
Why this change? I think Node 8.3+ supports the spread operator, and our `devEngines` block specifies 8.5+ or 9+

Reply:
@bvaughn Reverted this. When I first began work on this PR, I was using Node 8.0 on my machine. Switching to 8.5 fixed the issue.